mesh/accountsettings/accountsettings.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS AccountSettings(
+	settingID INT PRIMARY KEY AUTO_INCREMENT,
+    accountID INT,
+    isVerified BOOL NOT NULL DEFAULT FALSE,
+    enableContentFilter BOOL NOT NULL DEFAULT FALSE,
```

Comment:
For clarity, can you please rename this to something like `hasContentFilterEnabled`? I think the `is`, `has`, `does`, etc. prefixes convey the `BOOL` typing of this column best.